### DISCUSSION TITLE
xDrip

### DISCUSSION THREAD

**Discussion by zsfrz** [NightscoutFoundation/xDrip#3811] [Q&A]
I am a researcher specializing in glucose biosensors. I have a question regarding the functionality of the xDrip software.
Is it possible to link a custom or non-commercial sensor to xDrip, or is the software exclusively compatible with commercial sensors such as those from Dexcom or Medtronic?

**Comment by Navid200** [comment]
Do you mean a custom sensor as well as electronics to produce a Bluetooth channel?
Or do you mean just a sensor?

xDrip as is cannot be linked to any new sensor/transmitter unless we add the ability.  
I cannot make any promises.  But, I suspect even those who can may need more info.
